packages/scripts/config/webpack.config.js

```diff
@@ -70,16 +84,51 @@ const config = {
 			},
 			{
 				test: /\.svg$/,
+				exclude: /node_modules/,
 				use: [ '@svgr/webpack', 'url-loader' ],
 			},
+			{
+				test: /\.(sc|sa|c)ss$/,
+				exclude: /node_modules/,
+				use: [
+					{
+						loader: MiniCSSExtractPlugin.loader,
+					},
+					{
+						loader: require.resolve( 'css-loader' ),
+						options: {
+							sourceMap: ! isProduction,
+						},
+					},
+					{
+						loader: require.resolve( 'postcss-loader' ),
+						options: {
+							ident: 'postcss',
+							plugins: postcssPlugins,
+						},
+					},
+					{
+						loader: require.resolve( 'sass-loader' ),
+						options: {
+							sourceMap: ! isProduction,
+						},
+					},
+				],
+			},
 		],
 	},
 	plugins: [
-		// The WP_BUNDLE_ANALYZER global variable enables a utility that represents bundle
-		// content as a convenient interactive zoomable treemap.
+		// The WP_BUNDLE_ANALYZER global variable enables a utility that represents
+		// bundle content as a convenient interactive zoomable treemap.
 		process.env.WP_BUNDLE_ANALYZER && new BundleAnalyzerPlugin(),
-		// WP_LIVE_RELOAD_PORT global variable changes port on which live reload works
-		// when running watch mode.
+		// MiniCSSExtractPlugin to extract the CSS thats gets imported into JavaScript.
+		new MiniCSSExtractPlugin( { esModule: false, filename: '[name].css' } ),
+		// MiniCSSExtractPlugin creates JavaScript assets for CSS that are
+		// obsolete and should be removed. Related webpack issue:
+		// https://github.com/webpack-contrib/mini-css-extract-plugin/issues/151
```

Comment:
Appears to have been closed in favor of:

https://github.com/webpack-contrib/mini-css-extract-plugin/issues/85

Reply:
I'll update it, thank you for double-checking.